tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/YugipediaCardMapper.java

```diff
@@ -188,6 +189,11 @@ protected List<String> toMonsterTypes(String typesString) {
 	 * @return enum value
 	 */
 	protected CardType toCardTypeEnum(final String cardType) {
-		return CardType.valueOf(cardType.toUpperCase());
+		try {
+			return CardType.fromValue(cardType.toLowerCase());
+		} catch (final IllegalArgumentException e) {
+			// should not happen
```

Comment:
This is not true, this can happen as it some yugipedia cards are non-game ones but still contains a `CardTable2` template. For example, tokens and command cards.

```suggestion
```